File: C#/Lv_0/n개 간격의 원소들.cs
using System;

public class Solution {
    public int[] solution(int[] num_list, int n) {
        int[] answer = new int[num_list.Length%n == 0 ? num_list.Length/n : num_list.Length/n+1];
        for(int i=0, j=0; i<num_list.Length; i+=n, j++) answer[j] = num_list[i];
        return answer;
    }
}

File: JavaScript/Lv_0/n개 간격의 원소들.js
function solution(num_list, n) {
    return num_list.filter((x,y) => y%n == 0);
}

// 더 간단한 방법, _는 사용하지 않는 변수라는 뜻
// function solution(num_list, n) {
//     return num_list.filter((_,i) => !(i%n));
// }